### PR TITLE
Generate templates docs

### PR DESCRIPTION
**Pull request by chasefleming**
Docs for: https://github.com/onflow/flow-cli/issues/1233

### CODE REVIEW

File: docs/tools/flow-cli/utils/generate.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- **Aliases**: `g`
+
+### Generate Contract Template
+- **Usage**: `flow generate contract <name>`
+- **Short Description**: Generate a new contract.
+- **Example**: `flow generate contract HelloWorld`
+- **Required Arguments**: 
+  - `name`: Name of the contract.
+- **Flags**: 
+  - **Directory**: Specifies the directory in which the contract files will be generated. If not provided, a default directory is used.
```
Comment (gregsantos) [inline docs/tools/flow-cli/utils/generate.md:23]: Should prob specify what the default directory used is

**Review by gregsantos — CHANGES_REQUESTED** [review]

**Comment by bthaile** [comment]
Are you going to end up making a "Generate templates" type doc for all flow-cli commands?

**Comment by chasefleming** [comment]
@bthaile There are docs for most other commands: https://developers.flow.com/tools/flow-cli 

Is this what you're referring to?

**Comment by chasefleming** [comment]
New PR here: https://github.com/onflow/docs/pull/453

**Pull request closed without merging**